GothicVRProject/GothicVR
# NPC breathing and blinking

# To test
1. Load Lab scene
2. Activate GVRSingletons --> FeatureFlags --> enableNpcEyeBlinking
3. Spawn an NPC and play his animation
4. After the animation you should see NPC blinking (too long, but it works already) and breathing

Boundary:
* The feature for breathing won't really work in main game as NPCs are always doing something.
* Blinking is behind a feature flag so that we can pause implementation and rework in the future.

![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/5d2359ba-ec86-4bdb-b7f0-a1a557446461)
